mypy/nodes.py

```diff
@@ -683,6 +683,10 @@ def __init__(self,
     def name(self) -> str:
         return self._name
 
+    @property
+    def is_module_level_func(self) -> bool:
+        return self.info == FUNC_NO_INFO
```

Comment:
@JukkaL my instinct was to "hide" the use of `FUNC_NO_INFO`. I'm not sure if that was a good instinct.

Reply:
Actually I think that this is unnecessary (see below).